node/src/components/consensus/protocols/zug.rs

```diff
@@ -199,6 +201,9 @@ where
     next_scheduled_update: Timestamp,
     /// The write-ahead log to prevent honest nodes from double-signing upon restart.
     write_wal: Option<WriteWal<C>>,
+    /// A map of random IDs -> tipmestamp of when it has been created, allowing to track
+    /// to which nodes we have asked for a request.
```

Comment:
It's complete nitpicking at this point, but I still think we shouldn't be mentioning nodes here at all - we only know the IDs of sent requests, not which nodes they were sent to.

Reply:
Done